k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/values.yaml

```diff
@@ -104,19 +109,21 @@ cortxRgw:
 
 # cortxHA allows configuring CORTX HA settings
 cortxHa:
-  haxService:
-    protocol: https
-    name: cortx-hax-svc
-    port: 22003
+  enabled: true
 
 #cortxHare allows configuring CORTX HARE settings
 cortxHare:
   haxDataEndpoints: []
   haxServerEndpoints: []
   haxClientEndpoints: []
+  haxService:
+    protocol: https
+    name: cortx-hax-svc
+    port: 22003
```

Comment:
HAX is part of Hare, right? That's how it looks to me, not sure why I originally added it to the HA section. Correct me if I'm wrong.